### PR TITLE
Recd card

### PR DESCRIPTION
**Pull request by zzchua**
Refactor UI Card component, created immersive item view, add senderUsername and senderPhotoUrl to sendRecd db, remove dead code

### CODE REVIEW

File: src/components/common/TextH2.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import { TextRegular } from '../common';
+import { ColorScheme } from '../../styles/constants';
+
+const style = { fontSize: 50, color: ColorScheme.black };
+
+class TextH2 extends Component {
+  render() {
+    return (
+      <TextRegular>
+        <Text style={style}>
```
[the remark this answers is not in the document]
Comment (zzchua) [reply src/components/common/TextH2.js]: This is the correct way to do this according to React Native docs. TextRegular is the font styling...

[the remark this answers is not in the document]
Comment (kurokuriboh) [reply src/components/common/TextH2.js]: cool

[the remark this answers is not in the document]
Comment (zzchua) [reply src/components/common/TextH2.js]: This is correct as per react native documentation. Text Regular has text style for font.

File: src/screens/SendRecdScreen.js [hunk 2]
```diff
@@ -118,6 +118,8 @@ class SendRecdScreen extends Component {
   sendRecdToUsers() {
     this.props.sendRecd(
       this.props.currentUid,
+      this.props.currentUsername,
+      this.props.currentPhotoUrl,
```
[the remark this answers is not in the document]
Comment (zzchua) [reply src/screens/SendRecdScreen.js]: Changed this

File: src/actions/AuthActions.js [hunk 3]
```diff
@@ -63,7 +63,7 @@ export const userLoggedIn = ({
       uid,
       displayName,
       email,
-      photoURL,
+      photoUrl: photoURL,
```
Comment (kurokuriboh) [inline src/actions/AuthActions.js]: Shouldnt this be `photoUrl: photoUrl`?

**Pull request merged**